platform/persistence/platform-persistence-core-api/src/main/java/org/codice/ddf/persistence/PersistentItem.java

```diff
@@ -142,6 +145,18 @@ public Date getDateProperty(String name) {
     return (Date) getProperty(name + DATE_SUFFIX);
   }
 
+  public void encodeBinaryProperties() {
```

Comment:
:exclamation: I have two concerns with this method.  

The first is that it is unsafe since it could be called multiple times and I think would fail after the first call.  It would also be confusing because the key would still have the binary suffix but would now have a string value.

My second concern is that this is not really a concern of this class.  There are many different ways to encode binary data.  I think this should be moved to to the store commands which have a concern on how to serialize the data.  Base64 decoding is already done in the store import command.  The store export command should do Base64 encoding to keep the command format symmetry in one package.  Another option is to create a GSON converter like this gist example to use in the commands.

https://gist.github.com/orip/3635246

If you do that, it might be useful to add to our GSON type adapters.

https://github.com/codice/ddf/blob/master/libs/gson-support/src/main/java/org/codice/gsonsupport/GsonTypeAdapters.java#L38